config/locales/flash.en.yml

```diff
@@ -0,0 +1,20 @@
+en:
+  community_success: "Community Created Successfully"
+  community_updated: "Community Updated Successfully"
+  community_destroy: "Community Destroyed Successfully"
+  fill_all_fields: "fill all required fields"
+  draft_success: "Draft Saved Successfully"
+  comment_create: "Comment Created Successfully"
+  empty_comment: "Empty Comments are not Accepted!"
+  banned_successfully: "The user '%{username}' has been banned successfully!"
+  join_community: "Joined Community Successfully 🫂"
+  left_community: "You have left the Community 💔"
+  post_save: "Post saved!"
+  post_unsave: "Post Unsaved!"
+  post_edit: "Post Edited successfully"
+  post_closed: "Post Discussion Closed Successfully"
+  post_destroyed: "Post Destroyed Successfully"
+  user_not_prest: "Username '%{username}' not present in Community"
+  admin_ban: "The user '%{username}' is Admin"
+  already_banned: "The user '%{username}' is already banned"
+  not_present_to_ban: "The user '%{username}' is not present in "
```

Comment:
can we group this as error messages and success messages? under notice

Reply:
grouped the flash messages based on category